sycl/include/CL/sycl/queue.hpp

```diff
@@ -93,13 +93,37 @@ class queue {
   /// with the context provided, using the device returned by the device
   /// selector.
   ///
-  /// @param SyclDevice is an instance of SYCL device.
+  /// @param SyclContext is an instance of SYCL context.
   /// @param DeviceSelector is an instance of SYCL device selector.
   /// @param AsyncHandler is a SYCL asynchronous exception handler.
   /// @param PropList is a list of properties for queue construction.
   queue(const context &SyclContext, const device_selector &DeviceSelector,
         const async_handler &AsyncHandler, const property_list &PropList = {});
 
+#ifndef __SYCL_DISABLE_QUEUE_CONSTRUCTORS_EXTENSION__
```

Comment:
This guard is not needed anymore as this is standard queue API now.
See https://github.com/KhronosGroup/SYCL-Docs/pull/72.

Please, remove from here and from the test.

Reply:
Fixed.